recipe/meta.yaml

```diff
@@ -125,6 +129,8 @@ outputs:
         - {{ pin_subpackage("gxx_" ~ ctng_target_platform) }}
         - {{ pin_subpackage("gfortran_" ~ ctng_target_platform) }}
         - {{ pin_subpackage("binutils_" ~ ctng_target_platform) }}
+      run:
+        - sysroot_{{ ctng_target_platform }}
```

Comment:
```suggestion
```

Reply:
why?

Reply:
This would clobber.

Reply:
clobber what?

Reply:
I don't think we package any version of gcc with the sysroot built by ctng anymore.

Reply:
It would clobber the files. This package uses all the files from the compiler and is not supposed to have any depenencies.

Reply:
ahhhh it copies out of host